hwdb/70-mouse.hwdb

```diff
@@ -316,6 +316,12 @@ mouse:usb:v17efp6045:name:Lenovo USB Laser Mouse:
 # model name. The usb vid/pid is the same for all those devices.
 # Until 3.19 is available, this list just has the Wireless PID entry.
 
+# Logitech Anywhere MX
+mouse:usb:v046dpc52b:name:Logitech Unifying Device. Wireless PID:1017:
+mouse:usb:v046dp1017:name:Logitech Anywhere MX:
+ MOUSE_WHEEL_CLICK_ANGLE=20
+ MOUSE_WHEEL_CLICK_COUNT=18
```

Comment:
no need for the click count if the click angle is a integer fraction of 360 anyway, please remove that line. Otherwise, looks good, thanks.

Reply:
done